R/process_observers.R

```diff
@@ -0,0 +1,170 @@
+#------------------------------------------------------------------------------#
+# Functions/workflows for cleaning and processing the observer field
+#------------------------------------------------------------------------------#
+
+#' Arranges and saves a new version of the observer table 
+#' @param observer_table observer table data.frame
+#' @importFrom dplyr arrange
+save_observer_table <- function(observer_table) {
+  observer_table <- observer_table %>% arrange(county, route)
```

Comment:
FYI, this can be simplified to:

```r
observer_table %>% 
  arrange(county, route) %>%
  write.csv( "inst/extdata/observer_conversion_table.csv", row.names = F)
```

Also, to future proof:

```r
save_observer_table <- function(observer_table, file = "inst/extdata/observer_conversion_table.csv") {
  observer_table %>% 
    arrange(county, route) %>%
    write.csv(out, row.names = FALSE)
}
```

Reply:
I recommend spelling out `FALSE` and `TRUE`, which accords with the [tidyverse style guide](https://style.tidyverse.org/syntax.html#logical-vectors).

`FALSE` and `TRUE` are reserved words in the R language, while `T` and `F` are not.